bin/cml/runner.js

```diff
@@ -270,10 +271,10 @@ const runLocal = async (opts) => {
   RUNNER = proc;
   ({ id: RUNNER_ID } = await cml.runnerByName({ name }));
 
-  if (parseInt(idleTimeout) > 0) {
-    const watcher = setInterval(async () => {
-      let idle = RUNNER_JOBS_RUNNING.length === 0;
+  const watcher = setInterval(async () => {
+    let idle = RUNNER_JOBS_RUNNING.length === 0;
 
+    if (RUNNER_TIMER >= idleTimeout) {
```

Comment:
```suggestion
    if (parseInt(idleTimeout) > 0 && RUNNER_TIMER >= parseInt(idleTimeout)) {
```

Reply:
`if (parseInt(idleTimeout) > 0 &&` is managed by not setting the timeout (thing that I already removed in [3720126](https://github.com/iterative/cml/pull/1009/commits/372012695580157219c09b3830cdb1ace658beac) and I have to put back)

parseInt is not needed the value is always an integer due to the coerce
```js
coerce: (val) =>
          /^-?\d+$/.test(val) ? parseInt(val) : timestring(val),
```